src/circleci/test.clj

```diff
@@ -110,9 +113,11 @@
   *initial-report-counters*.  Returns the final, dereferenced state of
   *report-counters*."
   ([ns] (test-ns ns (constantly true)))
-  ([ns selector]
+  ([ns selector] (test-ns (read-config!) ns selector))
+  ([config ns selector]
```

Comment:
It's a bit unusual to have an optional arg at the front of the arglist; `config` here should probably get moved to the end.